### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by jonathanhora**
- [DEMO LINK](https://jonathanhora.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://jonathanhora.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **CSS Variable** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -15,8 +15,99 @@
...
+              href="#"
+              class="Link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="Link"
```
Comment (lerastarynets) [inline src/index.html:80]:
```suggestion
              class="link"
```
don't capitalize class names
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Nice start! Let's fix the solution according to the checklist

File: src/index.html (hunk 1 above)
Comment (jonathanhora) [reply src/index.html]: @lerastarynets , I made the changes

Comment (jonathanhora) [reply src/index.html]: @lerastarynets something is missing?

**Pull request closed without merging**